src/common/hashes/hashprefix.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Prefix for hashing functions.
+ *
+ * These prefixes are inserted before the source material used to
+ * generate various hashes. This is done to put each hash in its own
+ * "space." This way, two different types of objects with the
+ * same binary data will produce different hashes.
+ *
+ * Each prefix is a 4-byte value with the last byte set to zero
+ * and the first three bytes formed from the ASCII equivalent of
+ * some arbitrary string. For example "TXN".
+ */
+
+// transaction plus signature to give transaction ID
+export const HASH_TX_ID = 0x54584E00 // 'TXN'
```

Comment:
Any reason to not basket these up into an enum? 

It'll make your client code easier to read, I think:
`Hash.TX_ID' rather than `HASH_TX_ID` and imports will be cleaner